cmd/config.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/lunarway/shuttle/pkg/git"
+	"github.com/lunarway/shuttle/pkg/ui"
+	"github.com/spf13/cobra"
+)
+
+func newConfig(uii *ui.UI, contextProvder contextProvider) *cobra.Command {
+	var envVarsToExclude []string
+
+	envCmd := &cobra.Command{
+		Use:   "config",
+		Short: "Display shuttle context information",
+		Run: func(cmd *cobra.Command, args []string) {
+			uii.SetContext(ui.LevelSilent)
+			environmentVariables := os.Environ()
+			shouldExclude := make(map[string]bool)
+
+			for _, envVarToExclude := range envVarsToExclude {
+				shouldExclude[envVarToExclude] = true
+			}
+
+			context, err := contextProvder()
+			if err != nil {
+				panic(err)
+			}
+
+			fmt.Fprintf(cmd.OutOrStdout(), "Version %v\n", version)
+			breakLine(cmd)
+
+			fmt.Fprintf(cmd.OutOrStdout(), "Environment: \n")
+			for _, envVar := range environmentVariables {
+				if shouldExclude[extractEnvirontmentVariableName(envVar)] {
+					continue
+				}
+
+				fmt.Fprintf(cmd.OutOrStdout(), "%v\n", envVar)
+			}
+			breakLine(cmd)
+
+			parsedPlan := git.ParsePlan(context.Config.Plan)
+
+			if parsedPlan.IsGitPlan {
+				fmt.Fprintf(cmd.OutOrStdout(), "Plan: \n%v %v", context.Config.Plan, parsedPlan.Head)
+			} else {
+				fmt.Fprintf(cmd.OutOrStdout(), "Plan: \n%v", context.Config.Plan)
```

Comment:
I think we should move this output up above the environment output.

Usually one would expect way more env variables so having the "small lists" on top might give a better experience instead of a wall of env vars and then a couple of lines at the end.